agent/src/capability/baseconfigstore.cpp

```diff
@@ -67,8 +68,9 @@ ReturnCode BaseConfigStore::readCapsFromDir(const std::string &dirPath)
     for (std::string file : files) {
         std::string filePath = dirPath + file;
 
-        if (readCapsFromFile(filePath) != ReturnCode::SUCCESS) {
+        if (isError(readCapsFromFile(filePath))) {
             log_warning() << "Could not parse a file in directory: " << filePath;
+            return ReturnCode::FAILURE;
```

Comment:
why don't we `return readCapsFromFile(filePath);` instead ? `readCapsFromFile` function has error logs already on failure.